src/main/fc/config.c

```diff
@@ -1297,6 +1289,34 @@ void validateAndFixGyroConfig(void)
         if(motorConfig()->motorPwmRate > maxEscRate)
             motorConfig()->motorPwmRate = maxEscRate;
     }
+
+    uint32_t gyroFrequencyNyquist = (1.0f / (samplingTime * gyroConfig()->gyro_sync_denom)) / 2;
+
+     if (gyroConfig()->gyro_soft_lpf_hz > gyroFrequencyNyquist) {
+         gyroConfig()->gyro_soft_lpf_hz = 0;
```

Comment:
I still think runtime disabling is the best option in terms of being as user friendly as possible with the current configurator tools